--- a/CMakeLists.txt.orig 2018-07-27 07:52:10.460430024 +0000
+++ b/CMakeLists.txt  2018-07-27 08:37:24.809012628 +0000
@@ -12,14 +12,7 @@
 #  which can be found in the LICENSE file in the root directory, or at
 #  http://opensource.org/licenses/BSD-2-Clause
 #
-if(APPLE)
-  # RPATH support for OS X was introduced in this version
-  cmake_minimum_required(VERSION 2.8.12)
-else()
-  # It is likely/possible that this version could be decreased, but this version
-  # was chosen to support the default version of CMake in CentOS 7
-  cmake_minimum_required(VERSION 2.8.11)
-endif()
+cmake_minimum_required(VERSION 3.1)
 
 project(Elemental C CXX)
 set(EL_VERSION_MAJOR 0)
@@ -392,10 +385,11 @@
 # --------------
 include(detect/Fortran)
 
-# Detect C++11
+# Detect C++14
 # ------------
-include(detect/CXX)
-set(CXX_FLAGS "${CXX_FLAGS} ${CXX14_COMPILER_FLAGS}")
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
 
 # Detect MPI
 # ----------

