diff --git a/sci-libs/scalapack/scalapack-2.0.2-r1.ebuild b/sci-libs/scalapack/scalapack-2.0.2-r1.ebuild
index 55d6aed..f86038b 100644
--- a/sci-libs/scalapack/scalapack-2.0.2-r1.ebuild
+++ b/sci-libs/scalapack/scalapack-2.0.2-r1.ebuild
@@ -22,6 +22,8 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
 src_prepare() {
+	cmake-utils_src_prepare
+
 	# gentoo fix to avoid collision with other scalapack
 	sed -i \
 		-e 's:BINARY_DIR}/scalapack.pc:BINARY_DIR}/refscalapack.pc:g' \
